### PR TITLE
Merge Xcode 12.2 into d16-8

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

**Review by spouliot — APPROVED** [review]
needs
* final Xcode location (pending installation)
* bump to API_DIFF (with d16-8 released this week)
but they can be in a different PR

**Review by tj-devel709 — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]

**Comment by spouliot** [comment]
API diff looks fine, i.e. expected changes

**Comment by dalexsoto** [comment]
what? They pass locally...

File: Make.config [hunk 1]
```diff
@@ -181,7 +181,7 @@ MIN_TVOS_SIMULATOR_VERSION=10.2
 EXTRA_SIMULATORS=com.apple.pkg.iPhoneSimulatorSDK10_3 com.apple.pkg.AppleTVSimulatorSDK10_2 com.apple.pkg.WatchSimulatorSDK3_2
 
 INCLUDE_IOS=1
-#INCLUDE_MAC=1
+INCLUDE_MAC=1
```
Comment (rolfbjarne) [inline Make.config:184]: 🥳

**Review by rolfbjarne — COMMENTED** [review]
There are a few breaking changes in the api diff in the `FileProvider` namespace for Xamarin.Mac, are those intentional?

**Comment by spouliot** [comment]
@rolfbjarne they are not breaking changes with the XM stable release d16-8, but we compare with a single commit that comes from a branch where xcode12[.1] were merged (and unreleased for XM). 

I'll double check one last time as we need to do this manually for the (API diff) release notes anyway.

**Comment by spouliot** [comment]
There's one that needs to be fixed.

```
Type Changed: FileProvider.NSFileProviderItem_Extensions

Removed method:

	public static Foundation.NSNumber GetFavoriteRank (this INSFileProviderItem This);
```

I'll prep a PR to fix it.

**Comment by mandel-macaque** [comment]
@spouliot you could push in this smae PR, we have issues with the restore in the mac tests, so we should not land just yet.

**Comment by spouliot** [comment]
@mandel-macaque yeah, I was not sure if it would be updated/fixed before I got to it :)

**Comment by spouliot** [comment]
correct API diff for XM
https://gist.github.com/spouliot/67d3227fb5bf8baa0fece833bc208485

**Comment by dalexsoto** [comment]
@rolfbjarne AWESOME! Thank you!!!!! ♥️

**Comment by dalexsoto** [comment]
build

**Pull request merged**